musicbot/bot.py

```diff
@@ -1352,12 +1352,12 @@ def _gen_embed(self):
         """Provides a basic template for embeds"""
         e = discord.Embed()
         e.colour = 7506394
-        e.set_footer(
-            text=self.config.footer_text, icon_url="https://i.imgur.com/gFHBoZA.png"
-        )
+        # e.set_footer(
+            # text=self.config.footer_text, icon_url="https://i.imgur.com/gFHBoZA.png"
+        # )
         e.set_author(
             name=self.user.name,
-            url="https://github.com/Just-Some-Bots/MusicBot",
+            url="https://www.karhu.fi/",
```

Comment:
I would appreciate if you wouldn't use our bot as means of spreading your website.

Reply:
Oh sorry! I was working on some small customization  and I thought I did this on a different branch. Apparently it affected this MR too. I will remove this commit now. My bad :sweat:

Reply:
Thanks :D

Reply:
No problem :) Now it is fixed, I removed the customization stuff..